docs/api/utils.mdx

```diff
@@ -4,6 +4,86 @@ description: This doc describes `jotai/utils` bundle.
 nav: 2.2
 ---
 
+## Table of Contents
+
+1. [atomWithStorage](#atomWithStorage)
+
+   The atomWithStorage function creates an atom with a value persisted in localStorage or sessionStorage for React or AsyncStorage for React Native.
+
+2. [atomWithObservable](#atomWithObservable)
+   
+   The atomWithObservable function creates an atom from a rxjs (or similar) subject or observable. Its value will be last value emitted from the stream.
+
+3. [useUpdateAtom](#useUpdateAtom)
+   
+   Use useUpdateAtom and write-only atoms to avoid re-render.
+
+4. [useAtomValue](#useAtomValue)
+   
+   Returns the value of the given atom.
+
+5. [atomWithReset](#atomWithReset)
+   
+   Creates an atom that could be reset to its initialValue with useResetAtom hook.
+
+6. [useResetAtom](#useResetAtom)
+   
+   Resets a Resettable atom to its initial value.
+
+7. [RESET](#RESET)
+   
+   Special value that is accepted by Resettable atoms created with atomWithReset, atomWithDefault or writable atom created with atom if it accepts RESET symbol.
+
+8. [useReducerAtom](#useReducerAtom)
+   
+   Use this hook to update an atom value with a reducer function.
+
+9.  [atomWithReducer](#atomWithReducer)
+    
+    This is a function to create an atom with an embeded reducer function to update the value.
+
+10. [atomWithDefault](#atomWithDefault)
+    
+    This is a function to create a resettable primitive atom. Its default value can be specified with a read function instead of a static initial value.
```

Comment:
```suggestion
    This is a function to create an overwritable primitive atom. Its default value can be specified with a read function instead of a static initial value.
```